### DISCUSSION TITLE
Domains don't resolve any more; did the project move or die?

### DISCUSSION THREAD

**Discussion by jeffstearns** [timmo001/home-panel#2319] [General]
The Home Panel docs point to https://home-panel.timmo.dev/docs/setup but that domain doesn’t resolve (using Google’s 8.8.8.8 DNS server) as of 2023-09-09.

I tried contacting the author at the email address he publishes here (contact@timmo.xyz), but the timmo.xyz domain doesn't resolve either:

`DNS Error: DNS type 'mx' lookup of [timmo.xyz](http://timmo.xyz/) responded with code NXDOMAIN Domain name not found: [timmo.xyz](http://timmo.xyz/) Learn more at https://support.google.com/mail/?p=BadRcptDomain`

Is the project dead?  Did it move?  I can't find a newer location for it.  thx

**Comment by jpd2023** [comment]
No answer, comes the firm reply...

**Reply by HACS-bank** [reply]
https://community.home-assistant.io/t/home-panel-a-web-frontend-for-controlling-the-home/62597/556 also suggests that this project is inactive :(
